pkgs/build-support/docker/default.nix

```diff
@@ -911,6 +911,8 @@ rec {
     , fakeRootCommands ? ""
     , enableFakechroot ? false
     , includeStorePaths ? true
+      # Generate a Nix DB inside the image. The same caveats as `buildImageWithNixDb` apply.
```

Comment:
```suggestion
```

Please move this to user documentation at `doc/build-helpers/images/dockertools.section.md`.